contracts/README.md

```diff
@@ -0,0 +1,55 @@
+
+### cw20 Samples
+* [cw20 Basic](https://github.com/CosmWasm/cw-plus/tree/main/contracts/cw20-base): This is a basic implementation of a cw20 contract.
+* [cw20 Pot](https://github.com/deus-labs/cw-contracts/tree/main/contracts/cw20-pot): This project demonstrates a basic smart contract utilizing cw20 contract. Collected cw20 tokens in smart contract's balance is released to a target address after token amount exceeds a specified amount set during instantiation.
+* [Atomic Swaps](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-atomic-swap): This is a contract that allows users to execute atomic swaps. It implements one side of an atomic swap.
+* [cw20 Bonding curve](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-bonding): This builds on the Basic cw20 interface as implemented in cw20-base. 
+* [cw20 Escrow](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-escrow): This is an escrow meta-contract that allows multiple users to create independent escrows. Each escrow has a sender, recipient, and arbiter. It also has a unique id (for future calls to reference it) and an optional timeout.
+* [cw20 Merkle Airdrop](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-merkle-airdrop): This is a merkle airdrop smart contract that works with cw20 token specification Mass airdrop distributions made cheap and efficient.
+* [Staking Derivatives](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-staking): This is a sample contract that releases a minimal form of staking derivatives. This is to be used for integration tests and as a foundation for other to build more complex logic upon.
+* [cw20 Streams](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-streams): This contract enables the creation of cw20 token streams, which allows a cw20 payment to be vested continuously over time.
+
+### cw721 Samples
+* [cw721 Basic](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-base): This is a basic implementation of a cw721 NFT contract.
+* [cw721 Fixed Price](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-fixed-price): This contract enables the creation of limited edition fixed price NFTs according to the cw721 token standard.
+* [cw721 Metadata Onchain](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-metadata-onchain): NFT creators may want to store their NFT metadata on-chain so other contracts are able to interact with it. With cw721-Base in CosmWasm, we allow you to store any data on chain you wish, using a generic extension: T.
+* [cw721-expiration](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-expiration): This contract enables the creation of NFTs that expire after a predetermined number of days.
+* [SBT](https://github.com/CosmWasm/cw-nfts/tree/main/contracts/cw721-non-transferable): This is an implementation of the SBT contract.
```

Comment:
This link brings me cw721 something. Is thie SBT?

Reply:
I think an SBT (Soulbound Token) is another name for a CW721 non-transferable token. But maybe adding a description would make it more understandable.

Reply:
Aha, Okay. Thank you.

Reply:
I changed the name from the original without really thinking about it because I wanted to use the word SBT.
It needs to either find out what SBT is or change the name back.